extensions/azurePublish/src/components/ChooseProvisionAction.tsx

```diff
@@ -0,0 +1,264 @@
+import styled from '@emotion/styled';
+import { ChoiceGroup, IChoiceGroupOption } from 'office-ui-fabric-react/lib/ChoiceGroup';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { FluentTheme, NeutralColors } from '@uifabric/fluent-theme';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+
+// ---------- Styles ---------- //
+
+const Root = styled.div`
+  height: 100%;
+  width: 100%;
+  display: grid;
+  grid-template-columns: 30% 1fr;
+  grid-template-rows: 1fr;
+`;
+
+const ChoicesPane = styled.div`
+  height: 100%;
+  width: 100%;
+`;
+
+const ContentPane = styled(Stack)`
+  border-left: 1px solid ${NeutralColors.gray30};
+  height: 100%;
+`;
+
+const Content = styled(Stack)`
+  padding: 20px;
+`;
+
+const Title = styled(Text)`
+  font-size: ${FluentTheme.fonts.xLarge.fontSize};
+  margin: 8px 0;
+`;
+
+const Summary = styled(Stack)`
+  margin: 8px 0;
+`;
+
+const Details = styled(Stack)`
+  margin: 10px 0;
+`;
+
+const Instruction = styled(Stack)`
+  margin: 10px 0;
+`;
+
+const InstructionTitle = styled(Text)`
+  font-size: ${FluentTheme.fonts.smallPlus.fontSize};
+  text-transform: uppercase;
+  margin: 8px 0;
+`;
+
+const InstructionDetails = styled.div`
+  margin: 10px 0;
+`;
+
+const ResourceTitle = styled(Stack)`
+  margin: 10px 0;
+`;
+
+const LearnMoreLink = styled(Link)`
+  user-select: none;
+`;
+
+// ---------- CreateActionContent ---------- //
+
+const CreateActionContent = () => {
+  return (
+    <Content>
+      <Title>{formatMessage('Create New Resources')}</Title>
+      <Summary>
+        <Text>
+          {formatMessage(
+            'Select this option when you want to provision new Azure resources and publish a bot. A subscription to Microsoft Azure is required.'
+          )}
+        </Text>
+      </Summary>
+      <Details>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 1')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>{formatMessage('Sign in to Azure.')}</Text>
+          </InstructionDetails>
+        </Instruction>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 2')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>
+              {formatMessage(
+                'Select tenant & subscription, enter resource group name and resource name, and select region.'
+              )}
+            </Text>
+          </InstructionDetails>
+        </Instruction>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 3')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>
+              {formatMessage(
+                'Review & create new resources. Once provisioned these resources will be available in your Azure portal.'
+              )}
+            </Text>
+          </InstructionDetails>
+        </Instruction>
+      </Details>
+      <LearnMoreLink>{formatMessage('Learn More')}</LearnMoreLink>
```

Comment:
needs an href right?

Reply:
I will add when design/docs has one.  No idea what to put here.